specs/ilike.toml

```diff
@@ -0,0 +1,20 @@
+# To run: venv/bin/cr8  run-spec [--action {setup|queries|teardown}] specs/ilike.toml localhost:4200
```

Comment:
Not sure if we want to start adding this to all specs.. wouldn't it be more visible if we extended the README instead?